gfx/UltimateCataclysm/pngs_normal_32x32/overmap/river/river.json

```
@@ -0,0 +1,71 @@
+[{
+  "id": "river_nw",
+  "fg": ["river_nw"],
```

```suggestion
  "fg": "river_nw",
```
You should be able to omit arrays if there is only element. Please try.